app/build.gradle

```diff
@@ -21,8 +21,9 @@ android {
 
 dependencies {
```

Comment:
Did u actually change the minimum version?  It should show up in the build.gradle

Reply:
Nope, no changes to files were needed weirdly enough